deep_qa/data/fields/label_field.py

```diff
@@ -0,0 +1,55 @@
+from typing import Dict, List, Union
+
+from overrides import overrides
+import numpy
```

Comment:
I prefer to avoid abbreviations wherever possible, so that someone doesn't have to know my import conventions in order to read my code.  The first time I saw `pd` in a piece of code I had no idea what I was looking at, and it still hurts readability, just to save 4 characters on a line that's already short, typically.

Yes, there are some conventions that are widespread enough that it's ok, and maybe `numpy as np` is one of them, but it's just three characters - I prefer to use the full name.